### PR TITLE
Add README to the top-layout theme. Further fix #33.

### LINKED ISSUE #33: What is the "top" part?
I see a few references to a "top" (sub?)theme at https://github.com/hupe1980/gatsby-theme-material-ui/tree/master/packages/gatsby-theme-material-ui. What is that?

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: packages/gatsby-theme-material-ui-top-layout/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This package is [used by the `gatsby-theme-material-ui`](../gatsby-theme-material-ui/README.md#top-layout) to prevent FOUC.
```
Comment (oliviertassinari) [inline packages/gatsby-theme-material-ui-top-layout/README.md:1]: It doesn't sound accurate. I believe it has nothing to do with FOUC but that it's about the top-level rendering, viewport, etc.

Comment (dandv) [reply packages/gatsby-theme-material-ui-top-layout/README.md:1]: I think @hupe1980 understands much better what this theme does.

Comment (hupe1980) [reply packages/gatsby-theme-material-ui-top-layout/README.md:1]: The theme had its own "FOUC" issue and was fixed with #8. But that had nothing to do with the SSR FOUC.  That's what the plugin is for

**Comment by hupe1980** [comment]
Thx

**Pull request merged**